web/satellite/src/utils/authToken.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+// AuthToken exposes methods to manage auth cookie
+export class AuthToken {
+    private static readonly tokenKeySuffix: string = '_tokenKey';
+    private static tokenKey: string = '_tokenKey';
+    
+    public static initialize(): void {
+        AuthToken.tokenKey = (document as any).location.hostname + AuthToken.tokenKeySuffix;
+    }
+
+    public static get(): string {
+        return AuthToken.getCookie(AuthToken.tokenKey);
+    }
+
+    public static set(tokenValue: string): void {
+        document.cookie = AuthToken.tokenKey + '=' + tokenValue + '; path=/';
+    }
+
+    public static remove(): void {
+        document.cookie = AuthToken.tokenKey + '=; path=/';
+    }
+
+    private static getCookie(cname: string): string {
+        let name: string = cname + '=';
+        let decodedCookie: string = decodeURIComponent(document.cookie);
```

Comment:
Do we also need to cast `document` as `any` here?

Reply:
we didn't set --strictNullChecks flag, so document or document.location both could be null

to prevent this error i can cast document to any or use smth like this: document!.location!.hostname + AuthToken.tokenKeySuffix;

Reply:
@VinozzZ maybe we should think about modifying our tsconfig, so if u will have some thoughts about it - please, share them with us. Thanks!